tests/utilities/test_xla_device_utils.py

```diff
@@ -42,3 +46,14 @@ def test_xla_device_is_a_tpu():
     device = xm.xla_device()
     device_type = xm.xla_device_hw(device)
     return device_type == "TPU"
+
+
+def test_result_returns_within_10_seconds():
+    """Check that pl_multi_process returns within 10 seconds"""
+
+    start = time.time()
+    result = utils.pl_multi_process(time.sleep)(25)
+    end = time.time()
+    elapsed_time = math.floor(end - start)
+    assert elapsed_time <= 10
```

Comment:
what is the reason after rounding down to accept 10, it means that also 10.9 is valid?

Reply:
yep. the extra time is for the instructions before and after `.join(10)`